lib/resources/example_json/r4_examples_allergy_intolerance.rb

```diff
@@ -93,7 +96,18 @@ module Resources
         },
         "asserter": {
           "reference": "#19775876"
-        }
+        },
+        "reaction": [
+          {
+            "id": "58297369",
+            "manifestation": [
+              {
```

Comment:
I think it would be nice if we can show the manifestation with coding rather than just the text. 
Just my preference.

Reply:
updated